modules/mysql/testcontainers/mysql/__init__.py

```diff
@@ -86,3 +107,14 @@ def get_connection_url(self) -> str:
         return super()._create_connection_url(
             dialect="mysql+pymysql", username=self.username, password=self.password, dbname=self.dbname, port=self.port
         )
+
+    def _transfer_seed(self) -> None:
+        if self.seed is None:
+            return
+        src_path = Path(self.seed)
+        dest_path = "/docker-entrypoint-initdb.d/"
+        with BytesIO() as archive, tarfile.TarFile(fileobj=archive, mode="w") as tar:
+            for filename in src_path.iterdir():
```

Comment:
Note the classic `src_path.walk()` is actually not available until Python 3.12 for PosixPath (linux).
This lib seems to support lower versions, so `Path.iterdir()` is a good alternative